_posts/2022-06-21-cockpit-272.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Cockpit 272
+author: root
```

Comment:
Uh... this must be a side effect of switching to use the upstream container in Podman instead of Toolbx. Whoops.

Reply:
This will be fixed in #541, as I found a way to figure out the user on the host from within a standard Podman container. Overlays are in current mounts, which is mentioned in `/etc/mtab`, so we can parse it out. It's a bit of a hack, but it's the only place where I saw the current host `$USER` "leaked".